### ISSUE TITLE
console appender broken for DEBUG -- Karma test failure

### ISSUE THREAD

**Issue by ajoslin103** [aurelia/logging-console#18]
TL;DR

after running 'karma test' and inspecting the console output in the chrome window that Karma opened -- there are no debug messages, only info, warn, & error

**I'm submitting a bug report**

* **Library Version:**
aurelia-logging: "^1.5.0"

**Please tell us about your environment:**
OS X 10.14.4

* **Node Version:**
v10.14.0

* **NPM Version:**
6.7.0

* **JSPM OR Webpack AND Version**
JSPM 0.16.53
(Running against local jspm install)

* **Browser:**
Chrome

* **Language:**
karma ?

**Current behavior:**
browser console output:
WARN[test logger]
INFO[test logger]
ERROR[test logger]

**Expected/desired behavior:**
WARN[test logger]
INFO[test logger]
ERROR[test logger]
DEBUG[test logger]

* **What is the motivation / use case for changing the behavior?**
To see DEBUG messages in the browser console

**Comment by EisenbergEffect** [comment]
@ajoslin103 Any ideas what might be the cause? Is this something that changed with a recent CLI update?

**Comment by ajoslin103** [comment]
I don't know.  I know that .debug() hasn't worked for a while but I can't say when it stopped working

I moved the bug/error report from the framework down to this library because it's the smallest place I can make it happen

And I didn't write the code that's making it happen so I have a stronger confidence in it :)

If I had to speculate I'd say it's a change to the underlying WebKit browser technology somehow...

In Chrome & Safari I can type: console.debug('test') and I get back undefined with no echo of 'test' -- and the eager evaluation of console. does show a debug function...

In FireFox  I can type: console.debug('test') and it echo's 'test' before I get back undefined

I'm sorry, I just don't know....

**Comment by EisenbergEffect** [comment]
@fkleuver has done a ton of work with Karma. Let's see if he just happens to know what's going on offhand before we dig in further. Might be something really simple.

**Comment by ajoslin103** [comment]
Well it couldn't get much simpler...  And I feel quite the simpleton as well.

In the browser inspectors, when in the 'console' view there is a filter wherein you can choose the messages to display by level -- at some point Safari and Chrome started defaulting to not include debug messages.  Both of them have options (Chrome: verbose, Safari: all) that will show debug messages when enabled.

Sorry to waste your time, 
Al;

**Comment by EisenbergEffect** [comment]
No waste at all. Always glad to not have a bug :) Good to know about the change in default behavior as well. Thanks @ajoslin103 !
